appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -0,0 +1,120 @@
+$location=$args[0]
```

Comment:
I choose names that accurately, specifically describe what the thing is (for variables, classes, etc) or does (for methods). Good names are one of the most important aspects of code that is readable, and code has to be readable so that you can maintain or enhance it. It's totally worthwhile to spend a couple of minutes to come up with a good name. We can talk more offline.